Aleksandr_Prokopovich/02.Git.local/02.Git.local.md

```diff
@@ -0,0 +1,77 @@
+  349  mkdir 02.Git.local
```

Comment:
Please use github markdown for code sections

Reply:
Could you please share update

Reply:
I didn't understand how to use Markdown Cheatsheet.

Reply:
As html just apply for section of your code or objects

![image](https://user-images.githubusercontent.com/8258487/206203645-33cf2d13-7948-4307-9913-b622a9d22ac3.png)

links: 
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

https://www.youtube.com/watch?v=eJojC3lSkwg